tests/consensus_tests/test_routing_token.py

```
@@ -0,0 +1,158 @@
+
+def create_snapshot(peer_uri):
+    r = requests.post(f"{peer_uri}/collections/{COLLECTION_NAME}/snapshots?wait=true")
+    assert_http_ok(r)
+    return r.json()["result"]["name"]
+
+
+def recover_snapshot_no_sync(peer_uri, snapshot_url):
+    # `no_sync` keeps the recovered data authoritative on this peer *without* removing
+    # or re-syncing the other replica, leaving the two replicas in a divergent (but
+    # both Active) state — exactly the situation the routing token addresses.
+    r = requests.put(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)